src/index.html

```diff
@@ -1,13 +1,69 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700& display=swap"
+      rel="stylesheet"
+    >
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="product" data-qa="card">
+      <div class="product__wrapper">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE iMac 27 &#34;"
+          class="product__img"
+        >
+      </div>
+      <div class="description">
+        <h1 class="description__name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h1>
+        <span class="description__code">
+          Код товара: 195434
+        </span>
+      </div>
+      <div class="rate">
+        <div class="rate__stars stars--4">
```

Comment:
```suggestion
        <div class="rate__stars rate__stars--4">
```
Если у тебя нет блока stars, то ты не можешь тут указывать его модификатор.